cmd/up.go

```diff
@@ -60,12 +60,15 @@ type UpContext struct {
 	Sy         *syncthing.Syncthing
 	ErrChan    chan error
 	cleaned    chan struct{}
-	remotePort int
 	success    bool
 }
 
 func (up *UpContext) remoteModeEnabled() bool {
-	return up.remotePort > 0
+	if up.Dev == nil {
```

Comment:
This should never happen, right?

Reply:
it could if  this is called during a cleanup and  up.Dev wasn't set.